code/src/store/api/appApi.ts

```diff
@@ -3,8 +3,8 @@ import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
 
 export const appApi = createApi({
   baseQuery: fetchBaseQuery({
-    baseUrl: "https://jsonplaceholder.typicode.com/", // example url
-    credentials: "include"
+    baseUrl: "http://localhost:5000/api",
```

Comment:
this value should be defined by environment, it won't be the same on development stage and on prod.

Reply:
Fixed